### PR TITLE
Plant very early release notes for next release

### PR DESCRIPTION
**Pull request by lucafavatella**
Supersedes #795
In scope of https://www.pivotaltracker.com/story/show/154920627

* [x] Split release notes in separate file - one per release
* [x] Integrate to current master including https://github.com/aeternity/epoch/pull/795#issuecomment-369234528
* [x] Integrate https://github.com/aeternity/epoch/pull/795#issuecomment-369284096

### CODE REVIEW

**Comment by lucafavatella** [comment]
CI failing shall be fixed by https://github.com/aeternity/epoch/pull/810

**Comment by lucafavatella** [comment]
The only CI job meant to be running for this PR and affected by this PR is linux_package - that passed. I pushed same branch as distinct name and CI is running at https://circleci.com/workflow-run/5c19de97-7193-4d1a-824d-ad5e842c076a

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Comes with a naming system, for developerability.
 
 Written in Erlang.
 
-Please see the latest [release notes](RELEASE-NOTES.md) if you want to run the software or just [follow the progress of the project in Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2124891).
+Please see the latest [release notes](docs/release-notes) if you want to run the software or just [follow the progress of the project in Pivotal Tracker](https://www.pivotaltracker.com/n/projects/2124891).
```
Comment (dincho) [inline README.md]: Let's use either relative (or prefer) or absolute paths. You're using absolute path in following change and relative here.

**Comment by lucafavatella** [comment]
> The only CI job meant to be running for this PR and affected by this PR is linux_package - that passed. I pushed same branch as distinct name and CI is running at https://circleci.com/workflow-run/5c19de97-7193-4d1a-824d-ad5e842c076a UPDATE: Passed.

I see two approvals. Merging.

**Pull request merged**